src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -285,8 +284,6 @@
                      Margin="{StaticResource SmallTopMargin}"
                      Text="{x:Bind Mode=TwoWay, Path=ViewModel.ExcludedApps}"
                      IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"
-                     Width="375"
-                     Height="160"
```

Comment:
nit: Removing height property produces single-line text box (was multi-line before). What about going with `MinHeight` maybe?

Reply:
If you press the return key does it expand to multiline and show all the inserted strings?

Reply:
Yes, it does

Reply:
If `MinHeight` doesn't have any negative side effect, it would be better since it will communicate right away the list is a multi-line, even if it's specified in the label, visuals help a lot.

Reply:
Agree with @stefansjfw

Reply:
@enricogior Fully agree.

Ideally, we'd move to a [TokenizingTextBox](https://docs.microsoft.com/en-us/windows/communitytoolkit/controls/tokenizingtextbox) to reduce errors and to make it even more clearer to users. Example: ((please ignore the autosuggestions)

![TokenizedTextBox](https://user-images.githubusercontent.com/9866362/96849765-558d5200-1456-11eb-9ad8-c4e2bfc7e739.gif)

But that would be something beyond this PR, just food for thought 👍